normandy/bundler/models.py

```diff
@@ -21,7 +21,10 @@ def ids(self):
 
     @property
     def hash(self):
-        recipe_hashes = '_'.join([recipe.content_hash for recipe in self.recipes])
+        hash_parts = []
+        for recipe in self.recipes:
+            hash_parts.extend(action.implementation_hash for action in recipe.actions.all())
```

Comment:
Maybe add a property to the ``Recipe`` class that returns a list or concatenated string of all the hashes for the actions in the recipe?